nmmo/core/realm.py

```diff
@@ -77,13 +77,15 @@ def reset(self, map_id: int = None):
     # EntityState and ItemState tables must be empty after players/npcs.reset()
     self.players.reset()
     self.npcs.reset()
-    assert EntityState.State.table(self.datastore).is_empty(), \
-        "EntityState table is not empty"
+    #assert EntityState.State.table(self.datastore).is_empty(), \
+    #    "EntityState table is not empty"
+    EntityState.State.table(self.datastore).reset()
```

Comment:
Can we take these out now?

Reply:
Removed the EntityState table reset. Still need to track down ItemState, though ...